rocky/reports/report_types/aggregate_organisation_report/report.py

```diff
@@ -419,3 +424,28 @@ def collect_system_specific_data(self, data, services, system_type: SystemType,
         report_data = {key: value for key, value in report_data.items() if value}
 
         return report_data
+
+
+def aggregate_reports(
```

Comment:
I dont really understand why this code is moved here since this code will be the same for every type of aggregate report.

Reply:
So that we can re-use it in the tests: else we would be copying this logic over to generate real data for multi-reports